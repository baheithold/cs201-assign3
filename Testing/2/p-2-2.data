9 6 244 ; 7 0 191 ; 7 6 160 ; 6 2 238 ; 9 7 2 ; 
0 0 272 ; 9 2 287 ; 9 2 393 ; 6 2 282 ; 6 5 410 ; 
3 2 209 ; 3 3 20 ; 1 5 77 ; 4 3 436 ; 3 5 193 ; 
3 8 444 ; 7 7 164 ; 9 0 245 ; 1 4 384 ; 8 7 482 ; 
2 8 339 ; 5 9 434 ; 2 6 399 ; 6 5 190 ; 1 0 276 ; 
1 6 349 ; 8 0 274 ; 0 6 91 ; 5 0 302 ; 4 5 495 ; 
2 4 104 ; 1 0 76 ; 0 9 104 ; 5 1 346 ; 7 8 42 ; 
3 9 46 ; 5 4 277 ; 1 7 376 ; 4 0 438 ; 9 7 395 ; 
5 1 9 ; 5 0 342 ; 6 6 413 ; 7 2 153 ; 7 7 350 ; 
7 5 49 ; 3 4 342 ; 0 2 98 ; 2 1 61 ; 0 6 117 ; 
3 6 248 ; 3 8 293 ; 1 3 390 ; 4 9 86 ; 3 9 276 ; 
0 1 383 ; 4 3 462 ; 7 3 435 ; 8 7 98 ; 4 4 252 ; 
0 3 42 ; 0 8 128 ; 4 1 233 ; 3 9 315 ; 4 5 71 ; 
5 1 396 ; 1 3 336 ; 6 1 20 ; 3 9 177 ; 8 1 245 ; 
3 6 472 ; 3 4 14 ; 4 9 67 ; 4 4 191 ; 9 5 96 ; 
2 3 280 ; 8 9 498 ; 9 0 239 ; 8 9 305 ; 9 7 259 ; 
7 9 69 ; 1 5 484 ; 3 2 297 ; 7 3 104 ; 5 1 138 ; 
8 2 379 ; 8 0 212 ; 0 1 450 ; 9 8 381 ; 6 9 399 ; 
7 8 460 ; 3 4 108 ; 5 4 321 ; 1 8 334 ; 2 6 314 ; 
6 7 479 ; 8 1 429 ; 9 3 149 ; 8 4 11 ; 7 3 341 ; 

