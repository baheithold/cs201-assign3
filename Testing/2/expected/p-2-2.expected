0: 9
1: 3(9)46 4(9)86 7(9)2
2: 0(3)42 5(7)49 8(4)11
3: 1(5)77 6(0)91
4: 2(1)61
weight: 465
